packages-ts/starknet-gauntlet-emergency-protocol/src/commands/sequencerUptimeFeed/deploy.ts

```diff
@@ -24,6 +25,13 @@ const validateOwner = async (input) => {
   return true
 }
 
+const validateClassHash = async (input) => {
```

Comment:
class hashes and addresses are both pedersen hashes so they have the same validation check